Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/scars.yml

```
@@ -0,0 +1,259 @@
+- type: marking
+  id: ArmTwistingLeft
+  bodyPart: LArm
+  markingCategory: LeftArm
+  speciesRestriction: [Human, Oni, Reptilian, SlimePerson, Arachnid, Tajaran, Felinid, Arachne, Shadowkin, Vulpkanin, IPC]
+  followSkinColor: false
+  sprites:
+  - sprite: _Impstation/Mobs/Customization/scars.rsi
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)